copy/copy_test.go

```diff
@@ -137,39 +127,31 @@ func TestCopyDevicesAndFifo(t *testing.T) {
 }
 
 func TestCopySingleFile(t *testing.T) {
-	t1, err := ioutil.TempDir("", "test")
-	require.NoError(t, err)
-	defer os.RemoveAll(t1)
+	t1 := t.TempDir()
 
 	apply := fstest.Apply(
 		fstest.CreateFile("foo.txt", []byte("contents"), 0755),
 	)
 
 	require.NoError(t, apply.Apply(t1))
 
-	t2, err := ioutil.TempDir("", "test")
-	require.NoError(t, err)
-	defer os.RemoveAll(t2)
+	t2 := t.TempDir()
 
-	err = Copy(context.TODO(), t1, "foo.txt", t2, "/")
+	err := Copy(context.TODO(), t1, "foo.txt", t2, "/")
 	require.NoError(t, err)
 
 	err = fstest.CheckDirectoryEqual(t1, t2)
 	require.NoError(t, err)
 
-	t3, err := ioutil.TempDir("", "test")
-	require.NoError(t, err)
-	defer os.RemoveAll(t2)
+	t3 := t.TempDir()
```

Comment:
I saw a couple of these, and I guess those were copy/paste errors; we create `t3`, but add a defer to remove `t2` (I guess CI will show if it was not intentional 😂)